app/javascript/components/CurriculaHub2.js

```diff
@@ -0,0 +1,92 @@
+﻿import React from "react"
+import PropTypes from "prop-types"
+import AgegroupCard from "./AgegroupCard"
+import { withStyles } from '@material-ui/core/styles'
+import homeComponentStyles from "../../assets/javascripts/jss/components/homeComponentStyles.js";
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Divider from '@material-ui/core/Divider';
+import { NavHashLink as NavHashLink } from 'react-router-hash-link';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Fab from '@material-ui/core/Fab';
+import Icon from '@material-ui/core/Icon';
+import AddIcon from '@material-ui/icons/Add';
+
+
+class CurriculaHub2 extends React.Component {
+
+  render () {
+    const { classes } = this.props;
+    const data =
+          [ {
+            agegroup: "Kindergarten to 2nd Grade",
+            url: "/curricula/kToTwo"
+          },{
+            agegroup:"3rd Grade to 5th Grade",
+            url: "/curricula/threeToFive"
+          },{
+            agegroup:"6th Grade to 8th Grade",
+            url:"/curricula/sixToEight"
+           }
+          ]
+
```

Comment:
<img width="1399" alt="Screen Shot 2020-04-23 at 9 27 22 PM" src="https://user-images.githubusercontent.com/24725583/80172097-4877cf00-85a9-11ea-95ac-aea4afb0b2c6.png">
neither the left or the right card lead to anything useful. i think that's a pretty annoying user experience. can we hide those for now only show the 3rd to 5th grade subject cards when you go to the hub?
<img width="1376" alt="Screen Shot 2020-04-23 at 9 28 31 PM" src="https://user-images.githubusercontent.com/24725583/80172122-63e2da00-85a9-11ea-95c7-c07d6e307254.png">